src/Analyzers/MSTest.Analyzers/RoslynAnalyzerHelpers/ISymbolExtensions.cs

```diff
@@ -54,5 +55,20 @@ public static SymbolVisibility GetResultantVisibility(this ISymbol symbol)
 
             return visibility;
         }
+
+        public static ITypeSymbol? GetReferencedMemberOrLocalOrParameter(this ISymbol? symbol) => symbol switch
```

Comment:
This one is not copied from Roslyn analyzers so I would put it under `Helpers` folder.